doc/src/manual/metaprogramming.md

```diff
@@ -419,9 +420,9 @@ this is an extremely useful tool for debugging macros):
 
 ```julia
 julia> ex = macroexpand( :(@sayhello("human")) )
```

Comment:
Perhaps clearer by using the `@macroexpand` macro?

```julia
julia> ex = @macroexpand @sayhello "human"
:((Main.println)("Hello, ","human"))
```

to avoid the extra quoting stuff?